how-to.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: page
+title: How-to
+permalink: /howto/
```

Comment:
A link is broken in `how-to-install-ruby.md`. 
The link points to `/welcome-how-to/#previewing-your-post-locally`
It should now point to `/howto/#previewing-your-post-locally`